code/ATMOSPHERICS/hvac/geyser.dm

```diff
@@ -0,0 +1,46 @@
+#define GEYSER_TEMP 40
+#define GEYSER_POWER 40000
+#define GEYSER_HEAT_RANGE 3
+#define GEYSER_WARM_RATE 25
```

Comment:
Is this a debug value?

With the temperature change being min(temperature, M.body_temp + 25*1.5), a persons temperature'll jump up instantly by 37.5 at most, and this doesn't take into account their thermal protection being a value between 1 and 0 currently.

Reply:
I based it off coffee, yes